src/views/docs-view/components/docs-root-landing/utils/get-element-text-content.ts

```diff
@@ -0,0 +1,15 @@
+import { Children, ReactChild, ReactElement } from 'react'
+
+/**
+ * Given a React element,
+ * Return the text content of that element.
+ */
+export function getElementTextContent(element: ReactElement) {
```

Comment:
No changes to this utility relative to #733, have just moved it to a separate file.